openTSNE/tsne.py

```diff
@@ -1798,6 +1841,18 @@ def __call__(
                 should_eval_error=should_eval_error,
             )
 
+            if optimize_for_alpha:
+                dof -= dof_lr * alpha_grad
```

Comment:
This is a new optimizer. Can we optimize with the existing delta-bar-delta optimizer? Let's look at some loss curves.